RandLAPACK/misc/rl_util.hh

```diff
@@ -419,5 +451,24 @@ void eat_lda_slack(
     delete [] work;
 }
 
+template <typename T>
+void householder_unpacking(
+    int64_t m,
+    int64_t n,
+    int64_t b_sz,
+    T* Q_compressed,
+    T* Q_decompressed,
+    T* T_mat
+) {
```

Comment:
Please write a comment explaining what this function does. (The one thing that isn't obvious is ``b_sz``.)

Reply:
It is a dead prototype. Removed.